applications/nrf5340_audio/overlay-mcuboot.conf

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_PCD_APP=y
+CONFIG_BOARD_ENABLE_CPUNET=n
+
+# Those kconfig should be moved from board file to prj.conf
+# We need reduce size of mcuboot
+CONFIG_NFCT_PINS_AS_GPIOS=n
+
+# I2S
+CONFIG_I2S=n # Zephyr I2S driver not used, we are using custom shim
+CONFIG_NRFX_I2S=n
+
+# Enable NRFX_CLOCK for ACLK control
```

Comment:
Since all the comments in here describes why the configs need to be enable.
But actually in here we are going to disable all the configs. So we need to update the comments.

Reply:
Update comments from enable to disable.